486_Project/Characters_Class.py

```diff
@@ -0,0 +1,64 @@
+class warrior:
+    """
+    warrior class for player
+    :return: none
+    """
+    def __init__(self):
+
+        self.stats = [7, 5, 5, 7]
```

Comment:
What do these numbers mean? ([7, 5, 5, 7])

Reply:
[strength, dexterity, intelligence, constitution]
 I should have had some comment there sooner but focused on the code writing and left the comments on the "back-burner". Moving forward my should be more commented.